easybuild/easyblocks/generic/cmakemake.py

```diff
@@ -83,6 +95,10 @@ def configure_step(self, srcdir=None, builddir=None):
                 srcdir = default_srcdir
 
         options = ['-DCMAKE_INSTALL_PREFIX=%s' % self.installdir]
+
+        if '-DCMAKE_BUILD_TYPE' not in self.cfg['configopts']:
+            options.append("-DCMAKE_BUILD_TYPE=%s" % self.cfg['buildtype'])
```

Comment:
we don't do it for any other option (like seperate build dir).

It will be clear which one wins, the command run is shown?